### PR TITLE
Update striping documentation

### PR DESCRIPTION
**Pull request by mauricelimaquaq**

### CODE REVIEW

**Comment by jonathonmcmurray** [comment]
@mauricelimaquaq is this PR still needed or can it be closed?

**Comment by mauricelimaquaq** [comment]
> @mauricelimaquaq is this PR still needed/are you still waiting on me to review it or can it be closed?

Yeap waiting for you to review this so that it can be merged. The change in documentation is with reference to the change I did in TorQ FSP not to have the config files being written or changed by code.

File: docs/utilities.md [hunk 1]
```diff
@@ -800,25 +800,32 @@ However, this can be easily resolved by using more hex values from the output by
 
 ##### $KDBCONFIG/process.csv
 
-The process file should contain the RDB instances by specifying a different **`port`** and **`procname`** column. Set the **`load`** column to **`${KDBCODE}/processes/rdb.q`**.
+The process file should contain the RDB instances by specifying a different **`port`** and **`procname`** column. Set the **`load`** column to its respective **`${KDBCODE}/processes/rdb{i}.q`** file.
```
Comment (jonathonmcmurray) [inline docs/utilities.md:803]: What's different in each of the `rdb{i}.q` files? I thought we just had separate rdbsub.csv files for each process?

Comment (mauricelimaquaq) [reply docs/utilities.md]: Oh yes thats right. I have made the changes.

**Review by jonathonmcmurray — APPROVED** [review]
LGTM

**Pull request merged**